ghostscript.yaml

```diff
@@ -0,0 +1,136 @@
+package:
+  name: ghostscript
+  version: 10.01.2
+  epoch: 0
+  description: Interpreter for the PostScript language and for PDF
+  copyright:
+    - license: AGPL-3.0-or-later
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - build-base
+      - autoconf
+      - automake
+      - cups-dev
+      - expat-dev
+      - freetype-dev
+      - gtk-2.0-dev
+      - jbig2dec-dev
+      - lcms2-dev
+      - libjpeg-turbo
+      - libpng-dev
+      - libtool
+      # - openjpeg-dev
+      - tiff-dev
+      - zlib-dev
+
+var-transforms:
+  - from: ${{package.version}}
+    match: \.
+    replace: ''
+    to: mangled-package-version
+
+pipeline:
+  - uses: fetch
+    with:
+      expected-sha512: 11887850d440b8bbd8b83b8b58df8c743cf94be8fdbcfbda1a5fce55ce7c6a963d94b41a4ebcf1ba48f64f8af01e76dec789e2711212b1f7a32f98c42d75e0ff
+      uri: https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/download/gs${{vars.mangled-package-version}}/ghostscript-${{package.version}}.tar.gz
+
+  - uses: patch
+    with:
+      patches: fix-sprintf.patch
+
+  - uses: patch
+    with:
+      patches: ghostscript-system-zlib.patch
+
+  - runs: |
+      rm -r jpeg libpng zlib tiff lcms2mt cups/libs jbig2dec \
+      freetype
+
+      # fix parallel builds
+      sed -i -e 's/ECHO_XE/ECHOGS_XE/g' \
+      	-e 's/^\($(GLOBJ)md5.$(OBJ) :.*\)/\1 $(ECHOGS_XE)/' \
+      	base/lib.mak
+      aclocal && autoconf --force
+
+      cd ./ijs
+      libtoolize --force && aclocal && autoconf && automake --add-missing
+
+  - runs: |
+      export CFLAGS="$CFLAGS -flto=auto"
+
+      # build ijs
+      cd ./ijs
+
+      ./configure \
+      	--build=$CBUILD \
+      	--host=$CHOST \
```

Comment:
I think these need to be 
```
        --host=${{inputs.host}} \
        --build=${{inputs.build}} \
```
i.e. https://github.com/chainguard-dev/melange/blob/58d5b8b3a565bb012ff37fde1c38e4b359e79c96/pkg/build/pipelines/autoconf/configure.yaml#L23C1-L24C36

I wonder if we should us the reusable autoconf pipeline instead?

i.e. like this https://github.com/wolfi-dev/os/blob/d46144988d2bb6f40c37a603285d2935f01c2941/tomcat-native.yaml#L44-L55

and for the others below?

Reply:
thanks @rawlingsj, fixed.